devel/irrlichtmt-mt57/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        minetest irrlicht 1.9.0mt9
+
+name                irrlichtmt-mt57
+revision            0
+conflicts           irrlichtmt-mt56
```

Comment:
Could you resolve this "conflict" and make it possible to install both ports on the same system? Conflicting ports make upgrades difficult.

I strongly recommend you to use the subport mechanics, because in this case most parts of the Portfiles are the same.

Reply:
@l2dy - THX - Good proposal, but **where to find applicable documentation, please?**

> I strongly recommend you to use the subport mechanics, because in this case most parts of the Portfiles are the same.

AFAIK the [Chapter 4. Portfile Development](https://guide.macports.org/#development) lacks sufficient documentary on the 'subport' excellent opportunities both for new Portfile development and port mainentance, unfortunately.

Ref https://github.com/macports/macports-guide/pull/53

Please, poor new maintainer **needs some advice.**

Reply:
However, I would like the idea of parallel installs and the option of parallel Apps to MT players and devs.

There will be fun ...

🌻

Reply:
@l2dy - I am somewhat lost by the explanations of Guide and would like to get advice from you, if possible.

Would a statement like `worksrcdir  ${subport}` change the a.m. path problem in an applicable way?

Reply:
Unfortunately, there is no official documentation on these topics. You need to learn from other ports that are doing this.

I don't have the time to guide you through this process now, so you are on your own if you insist splitting the port.

Just to be clear, in order to ensure future upgrades work correctly, I object merging this PR as is.

Reply:
> I don't have the time to guide you through this process now, so you are on your own if you insist splitting the port.

Again some misunderstanding and I wonder why you insist on old news and ignore my changed proposal apparently. IMO you should avoid being stressed by such free and open endeavour. Warmest wishes.

> Unfortunately, there is no official documentation on these topics. You need to learn from other ports that are doing this.

Understood. However, there shall be **subports** in one Portfile to comprise in one port accordingly.
Would allow future parallel installs and the option of parallel Apps to MT players and devs.

This is fun.

🌻

Reply:
Anyway, no need for any hurries.

This PR from start was marked as DRAFT and will be WIP until 'minetest' is more integrated.